templates/myaccount/dashboard.php

```diff
@@ -14,19 +14,19 @@
  *
  * @see         https://docs.woocommerce.com/document/template-structure/
  * @package     WooCommerce/Templates
- * @version     2.6.0
+ * @version     4.3.0
  */
 
 if ( ! defined( 'ABSPATH' ) ) {
-	exit; // Exit if accessed directly
+	exit; // Exit if accessed directly.
 }
 ?>
 
 <p>
 	<?php
 	printf(
 		/* translators: 1: user display name 2: logout url */
-		__( 'Hello %1$s (not %1$s? <a href="%2$s">Log out</a>)', 'woocommerce' ),
+		esc_html__( 'Hello %1$s (not %1$s? <a href="%2$s">Log out</a>)', 'woocommerce' ),
```

Comment:
While it's correct that the text should be escaped, this will not render the link to log out, but the `<a>` tag as text:

![Screenshot 2020-06-11 at 13 41 07](https://user-images.githubusercontent.com/2207451/84381361-3994bc80-abe9-11ea-8f00-1f00cf65283e.png)

Reply:
Oops. Thank you! I fixed this.